sdk/python/vectors.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-python3 -m tests.vectors.all --blockchain nem --vectors ../test-vectors/nem
-python3 -m tests.vectors.all --blockchain symbol --vectors ../test-vectors/symbol
+set -ex
+
+python3 -m tests.vectors.all --blockchain nem --vectors "$(git rev-parse --show-toplevel)/tests/vectors/nem"
+python3 -m tests.vectors.all --blockchain symbol --vectors "$(git rev-parse --show-toplevel)/tests/vectors/symbol"
```

Comment:
why/how was it working  earlier? was it downloading vectors anywhere?

Reply:
it was not working in monorepo. previously, travis was downloading vectors.